weblinc-js-adapter.sublime-snippet

```diff
@@ -0,0 +1,109 @@
+<snippet>
+  <content><![CDATA[
+/**
+ * @namespace WEBLINC.${1:adapterName}
+ */
+WEBLINC.analytics.registerAdapter('${1:adapterName}', function () {
+    'use strict';
+
+    var pageViewSent = false,
+
+        send = function (event, payload) {
+            if (WEBLINC.analytics.debug) {
+                console.log(arguments);
+            }
+
+            ${2://add your send logic here}
+        },
+
+        isLoaded = function () {
+            var scripts = _.toArray(document.scripts),
+
+            loaded = _.any(scripts, function (script) {
+                return script.src.match(/someregexthatmatchesyour3rdpartyscript/i);
```

Comment:
`someregexthatmatchesyour3rdpartyscript` should be `${3:someregexthatmatchesyour3rdpartyscript}`